vaas-app/src/vaas/cluster/cluster.py

```diff
@@ -25,16 +26,32 @@ def make_parallel_loader(max_workers=settings.VAAS_LOADER_MAX_WORKERS,
     else:
         return ParallelLoader(max_workers)
 
-
 @app.task(bind=True, soft_time_limit=settings.CELERY_TASK_SOFT_TIME_LIMIT_SECONDS)
 def load_vcl_task(self, emmit_time, cluster_ids):
+    emmit_time_aware = timezone.make_aware(datetime.strptime(emmit_time, "%Y-%m-%dT%H:%M:%S.%fZ"),
+                                           timezone=timezone.utc)
+    if settings.STATSD_ENABLE:
+        total_processing_load_vcl_task_time = timezone.now() - emmit_time_aware
+        statsd.timing('queue_time_from_order_to_execute_task', total_processing_load_vcl_task_time)
+
     start_processing_time = timezone.now()
     clusters = LogicalCluster.objects.filter(pk__in=cluster_ids, reload_timestamp__lte=emmit_time)
     if len(clusters) > 0:
-        return VarnishCluster().load_vcl(start_processing_time, clusters)
+        varnish_cluster_load_vcl = VarnishCluster().load_vcl(start_processing_time, clusters)
```

Comment:
We should have more detailed metrics about vcls rendering times and loading it to Varnishes times

Reply:
In the near future i think we can discuss what kind of additional metrics we wish to have.